core/src/main/kotlin/org/evomaster/core/output/ExpectationsWriter.kt

```diff
@@ -14,6 +14,7 @@ class ExpectationsWriter {
     private val responseStructureOracle = "responseStructureOracle"
     private lateinit var swagger: Swagger
     private lateinit var partialOracles: PartialOracles
+    private val portRegex = """\w+:\d{4,5}""".toRegex()
```

Comment:
this would not catch `localhost:80` for example. i guess `{1,5}` would be better

and what about `127.0.0.1:8080`? that would not be considered.

i guess could have something like: `(\w+|(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})):\d{1,5}`

Reply:
Agreed. Although I'm still hoping to think of a better and more long-term solution than this.